vendor/github.com/google/cadvisor/manager/manager.go

```diff
@@ -242,7 +242,7 @@ func retryDockerStatus() info.DockerStatus {
 	for {
 		ctx, _ := context.WithTimeout(context.Background(), startupTimeout)
 		dockerStatus, err := docker.StatusWithContext(ctx)
-		if err == nil {
+		if err != nil {
```

Comment:
this is reverting https://github.com/openshift/origin/pull/19601/files, drop this change